frontend/src/components/Sidebar.less

```diff
@@ -0,0 +1,34 @@
+.sidebar {
+    display: flex;
+    flex-direction: column;
+
+    min-height: 100%;
+
+    .header {
+        flex: 0 0;
+
```

Comment:
why do we need extra spaces everywhere in this file?

Reply:
css properties are grouped and groups are split by a new line. That way it's easier to read. (at least for me)

Reply:
did we decide to do so in this project?
If we did so, we should add a linter to ensure the proper order and grouping for the shake of the consistency.

Reply:
I don't know any linter rules which do that. I also don't know how to formalize it. We agreed to follow similar practices as we did in code-annotation. And we did it in code annotation: https://github.com/src-d/code-annotation/blob/master/src/components/Breadcrumbs.less

Reply:
I do not like to follow complex rules that are not programmatically enforced.
To do so, I think we could try to configure a linter like [stylelint.io](https://stylelint.io) + [stylelint-order](https://github.com/hudochenkov/stylelint-order), or analyze if we can use [ConcentricCSS](https://github.com/brigade/scss-lint/blob/master/data/property-sort-orders/concentric.txt) with less.

Otherwise, we're having inconsistencies as in https://github.com/src-d/code-annotation/blob/master/src/components/Experiment/Diff.less#L61

Reply:
1. I don't think any linter should be enforced for css blocks with <=3 rules or even 4-5 in some cases. Making it 6 lines instead of 3 would make it ugly and more difficult.
2. Using new line to split css rules to groups is quite common practice. For example:
- [Foundation UI](https://github.com/zurb/foundation-sites/blob/develop/scss/components/_switch.scss#L91)
- [Semantic UI](https://github.com/Semantic-Org/Semantic-UI/blob/master/src/definitions/elements/button.less#L24)
as you could notice they also don't apply any grouping rules for classes with only a few lines
3. I thought a bit about how I split the groups. I have main groups like:
    1. layout mode: position/display/flex/...
    2. layout dimensions: width/height/margin/padding/...
    3. decoration: background/blox-shadow/...
    4. text: color/font-size/text-tranform/...
and use them when that makes sense. (usually when a group has more than 2 rules with layout mode as an exception)

If I didn't convince you, please be welcome to reformat it in your own style as soon as you start writing css for this project.